### PR TITLE
doc: Change README_windows to Markdown

### PR DESCRIPTION
**Pull request by cvengler**
To make it more coherent with the other README files in the doc/ folder

### CODE REVIEW

**Comment by fanquake** [comment]
Please fix any trailing whitespace issues, see linter output [here](https://travis-ci.org/bitcoin/bitcoin/jobs/423605915).

You'll also need to update any instances where this file is used, such as in [share/setup.nsi.in](https://github.com/bitcoin/bitcoin/blob/271b379e636afa419c5208cb462c07090490266c/share/setup.nsi.in#L79) or [Makefile.am](https://github.com/bitcoin/bitcoin/blob/77168f766f15c6621fb2af0dfcf2dd12b971172c/Makefile.am#L56).

I'm wondering if there was a Windows specific reason the readme was left as a `.txt` file?

**Comment by ken2812221** [comment]
I am not sure it is good to fix the linter issue. When we change `\r\n` to `\n`, it will show just one line if it is opened by Windows built-in notepad.

**Comment by emilengler** [comment]
@fanquake Thanks for that reminder I updated it. I also done `grep -rn README_windows.txt *` to be 100% sure.

**Comment by ken2812221** [comment]
I am NACK about the line ending change thing. IMO it just need to add an exception into the linter not to check `\r\n`.

**Comment by laanwj** [comment]
NACK

Leave it `.txt` please. This file is intended to be opened in notepad, changing the extension/layout does nothing to help windows users.

File: doc/README_windows.md [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
-However, it downloads and stores the entire history of Bitcoin transactions;
-depending on the speed of your computer and network connection, the synchronization
-process can take anywhere from a few hours to a day or more.
-
-See the bitcoin wiki at:
-  https://en.bitcoin.it/wiki/Main_Page
-for more help and information.
+# Bitcoin Core
+
+### Intro
```
Comment (practicalswift) [inline doc/README_windows.md]: ```doc/README_windows.md:3: MD001 Header levels should only increment by one level at a time```

File: doc/README_windows.md [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
+Bitcoin is a free open source peer-to-peer electronic cash system that is
+completely decentralized, without the need for a central server or trusted
+parties.  Users hold the crypto keys to their own money and transact directly
+with each other, with the help of a P2P network to check for double-spending.
+
+
```
Comment (practicalswift) [inline doc/README_windows.md]: ```doc/README_windows.md:9: MD012 Multiple consecutive blank lines```

File: doc/README_windows.md [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
+### Setup
+Unpack the files into a directory and run bitcoin-qt.exe.
+
+Bitcoin Core is the original Bitcoin client and it builds the backbone of the network.
+However, it downloads and stores the entire history of Bitcoin transactions;
+depending on the speed of your computer and network connection, the synchronization
+process can take anywhere from a few hours to a day or more.
+
+See the bitcoin wiki [here](https://en.bitcoin.it/wiki/Main_Page) for more help and information.
```
Comment (practicalswift) [inline doc/README_windows.md]: ```doc/README_windows.md:18: MD013 Line length```

File: doc/README_windows.md (hunk 1 above)
Comment (practicalswift) [reply doc/README_windows.md]: ```doc/README_windows.md:3: MD022 Headers should be surrounded by blank lines```

File: doc/README_windows.md [hunk 4]
```diff
@@ -1,23 +1,18 @@
...
+
+### Intro
+Bitcoin is a free open source peer-to-peer electronic cash system that is
+completely decentralized, without the need for a central server or trusted
+parties.  Users hold the crypto keys to their own money and transact directly
+with each other, with the help of a P2P network to check for double-spending.
+
+
+### Setup
+Unpack the files into a directory and run bitcoin-qt.exe.
```
Comment (practicalswift) [inline doc/README_windows.md]: ```doc/README_windows.md:10: MD022 Headers should be surrounded by blank lines```

File: doc/README_windows.md [hunk 5]
```diff
@@ -1,23 +1,18 @@
...
-depending on the speed of your computer and network connection, the synchronization
-process can take anywhere from a few hours to a day or more.
-
-See the bitcoin wiki at:
-  https://en.bitcoin.it/wiki/Main_Page
-for more help and information.
+# Bitcoin Core
+
+### Intro
+Bitcoin is a free open source peer-to-peer electronic cash system that is
```
Comment (practicalswift) [inline doc/README_windows.md]: ```doc/README_windows.md:3: MD022 Headers should be surrounded by blank lines```

**Pull request closed without merging**